packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/Camera2CameraInfoHostApiImpl.java

```diff
@@ -53,14 +54,22 @@ public Long getSensorOrientation(@NonNull Camera2CameraInfo camera2CameraInfo) {
       return Long.valueOf(
           camera2CameraInfo.getCameraCharacteristic(CameraCharacteristics.SENSOR_ORIENTATION));
     }
+
+    @NonNull
+    @OptIn(markerClass = ExperimentalCamera2Interop.class)
```

Comment:
Shouldn't need this `OptIn` because the annotation is on the entire class.

Reply:
Removed.